documentation/bolt_installing_modules.md

```diff
@@ -1,28 +1,32 @@
 # Installing modules
 
-To share the Bolt plans and tasks that you've created on the Forge, you can
-package them into Puppet modules. If you'd like to use a plan or task from a
-module that you found on the Forge, you can use Bolt to install the module.
+When you use the Bolt command line to install a module, Bolt manages your
+dependencies for you. 
 
-## 🧪 New experimental module management
+You can use the command line to:
+- Create a project with pre-installed modules.
+- Add a Puppet Forge module to your existing Bolt project.
+- Install the modules associated with a Bolt project. This is useful if you've
+  downloaded a project from source control and want to install its dependencies.
+- Update your project's modules.
 
-**Experimental**: Bolt 2.30.0 introduced changes to how modules are managed in
-Bolt projects. To find out how to opt in to the changes, see [Managing
-modules](managing_modules.md).
+If you need to install a module from a GitHub repository or an alternate Forge,
+or you need to use a Forge proxy, you can manually configure your Bolt modules
+in your Bolt project configuration file (`bolt-project.yaml`).
 
-## Create a new Bolt project and install a list of modules with dependencies
+## Create a Bolt project with pre-installed modules
 
 If you want to get started with a new Bolt project and you need specific modules
 from the Puppet Forge, you can install the modules and their dependencies using
 the Bolt command line.
 
-When you create a new project with the `modules` command-line option and a
+When you create a project with the `modules` command-line option and a
 comma-separated list of Forge modules, Bolt installs the latest versions of each
 module and resolves and installs all dependencies required by those modules. For
-example, to create a project named `example_project` with the `apache` and
-`mysql` modules installed, use the following command:
+example, to turn the current directory into a project named `example_project`
+with the `puppetlabs/apache` and `puppetlabs/mysql` modules installed, use the following command:
 
-- _\*nix shell command_
+_\*nix shell command_
```

Comment:
I can't add a suggestion, unfortunately, but the PowerShell cmdlet a couple lines down should also be un-bulleted.